controllers/servers/host_definer/common/settings.py

```diff
@@ -0,0 +1,37 @@
+STORAGE_API_VERSION = 'storage.k8s.io/v1'
+CSINODE_KIND = 'CSINode'
+CSI_IBM_BLOCK_API_VERSION = 'csi.ibm.com/v1'
+CSI_IBM_BLOCK_GROUP = 'csi.ibm.com'
+VERSION = 'v1'
+HOSTDEFINITION_PLURAL = 'hostdefinitions'
+HOSTDEFINITION_KIND = 'HostDefinition'
+SECRET_NAME_KEY = 'secret_name'
+SECRET_NAMESPACE_KEY = 'secret_namespace'
+SECRET_NAME_SUBSTRING = 'secret-name'
+DEPRECATED_SECRET_NAME_SUBSTRING = 'SecretName'
+SECRET_NAME_KEY_IN_STORAGE_CLASS = 'csi.storage.k8s.io/secret-name'
+SECRET_NAMESPACE_KEY_IN_STORAGE_CLASS = 'csi.storage.k8s.io/secret-namespace'
+IBM_BLOCK_CSI_DRIVER_NAME = 'block.csi.ibm.com'
+MANAGEMENT_ADDRESS_KEY = 'management_address'
+USERNAME_KEY = 'username'
+PASSWORD_KEY = 'password'
+ADDED_EVENT = 'ADDED'
+DELETED_EVENT = 'DELETED'
+MODIFIED_EVENT = 'MODIFIED'
+PENDING_PHASE = 'Pending'
+PENDING_CREATION_PHASE = 'PendingForCreation'
+PENDING_DELETION_PHASE = 'PendingForDeletion'
+ERROR_PHASE = 'Error'
+READY_PHASE = 'Ready'
+CSI_IBM_BLOCK_PRODUCT_LABEL = 'product=ibm-block-csi-driver'
+IBM_BLOCK_CSI_NODE_PREFIX = 'ibm-block-csi-node'
+BINARY_VALUE_FOR_DELETED_POD = 0
+BINARY_VALUE_FOR_EXISTING_POD = 1
```

Comment:
why binary and not just use hardcoded boolean values?

Reply:
removed them